pkgs/applications/window-managers/evilwm/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl,  libX11, libXext, libXrandr, libXrender,
+  xproto, xextproto, randrproto, renderproto, kbproto,  patches ? [] }:
+
+stdenv.mkDerivation rec {
+  name = "evilwm-1.1.1";
+
+  src = fetchurl {
+    url = "http://www.6809.org.uk/evilwm/${name}.tar.gz";
+    sha256 = "79589c296a5915ee0bae1d231e8912601fc794d9f0a9cacb6b648ff9a5f2602a";
+  };
+
+  buildInputs = [ libX11 libXext libXrandr libXrender
+                  xproto xextproto randrproto renderproto kbproto ];
+
+  prePatch = ''sed -i "s@/usr@$out@" Makefile'';
```

Comment:
Can you link me a description of this, I don't quite understand. `sed -i` with alternate delimiters seems to work on darwin for several packages, it is used quite liberally throughout so I'd expect this issue to have come up before ...